common/tier4_screen_capture_rviz_plugin/src/screen_capture_panel.cpp

```diff
@@ -99,11 +119,27 @@ void AutowareScreenCapturePanel::on_rate_change([[maybe_unused]] const int rate)
 {
   timer_->cancel();
   create_timer();
+  update_buffer_size();
   RCLCPP_INFO_STREAM(raw_node_->get_logger(), "RATE:" << rate_->value());
 }
 
+void AutowareScreenCapturePanel::on_buffer_size_change(const int buffer_size)
+{
+  update_buffer_size();
+  RCLCPP_INFO_STREAM(
+    raw_node_->get_logger(),
+    "BUFFER SIZE: " << buffer_size << " [sec] with " << buffer_size_frames_ << " frames)");
+}
+
+void AutowareScreenCapturePanel::update_buffer_size()
+{
+  // Calculate buffer sizes in frames
+  buffer_size_frames_ = buffer_size_->value() * rate_->value();
+}
+
 void AutowareScreenCapturePanel::on_timer()
 {
+  // update_buffer_size();
```

Comment:
@kyoichi-sugahara Is this necessary diff...?

Reply:
@satoshi-ota 
no :cry: 
Thank you for checking!
Addressed in https://github.com/autowarefoundation/autoware_tools/pull/232/commits/4ed7b1d524114c54f0e1fa2c4dc01f44b750ed8f